R/util.R

```diff
@@ -81,18 +81,24 @@ defaults = function(x, defaults) {
 # workaround replacements for other patterns that don't quite do what we need them to
 # (especially when it comes to rvars...)
 
-#' @importFrom dplyr bind_rows
-#' @importFrom vctrs vec_chop
+map_dfr_ = function(data, fun, ...) {
+  # drop-in replacement for purrr::map_dfr
+  do.call(
```

Comment:
bind_rows can take a list so should be able to write this without `do.call()`, something like `bind_rows(lapply(data, fun, ...))`

Also could you add the imports back in? When functions are used several times I prefer importing them over using `::`. I find it more readable. Thanks!

Reply:
`vctrs::vec_chop()` is only used in this one spot. Want it imported?

Reply:
Good point, nah can leave it using `::`